linux_os/guide/services/usbguard/usbguard_allow_hid/oval/shared.xml

```diff
@@ -1,14 +1,6 @@
 <def-group>
   <definition class="compliance" id="usbguard_allow_hid" version="1">
-    <metadata>
-      <title>Check that USB Human Interface Devices are allowed by USBGuard rules</title>
-      <affected family="unix">
-        <platform>multi_platform_fedora</platform>
-        <platform>Red Hat Enterprise Linux 8</platform>
-        <platform>Oracle Linux 8</platform>
-      </affected>
-      <description>Check that /etc/usbguard/rules.conf contains at least one non white space character empty and exists.</description>
-    </metadata>
+    {{{ oval_metadata("Check that /etc/usbguard/rules.conf contains at least one non white space character empty and exists.") }}}
```

Comment:
me neither, but according to the rest of the OVAL, it's probably meant to be
```suggestion
    {{{ oval_metadata("Check that /etc/usbguard/rules.conf exists and that it contains at least one non white space character.") }}}
```
Speaking of which, this doesn't seem to be in line with the rule description, but OTOH it's just a convenience feature.
@vojtapolasek do you remember why this usbguard `rules.conf` check doesn't check for an occurrence of a specific string?

Reply:
Hi, it does not because it is a convenience feature and we don't want to impose restrictions on administrators. They can prohibit / allow whatever they want there.

Reply:
OK, fair enough!